src/sections/community/Community.Hero.tsx

```diff
@@ -1,15 +1,34 @@
-import Image from "next/image";
+import Link from "next/link";
+import { useState } from "react";
 
 import CommunitiesGrid from "src/components/CommunitiesGrid";
 import ScrollDownArrow from "src/components/ScrollDownArrow";
+import CommunitySubOrgs from "./Community.Suborgs";
 
 const CommunityHero: React.FC<{ isMobile: boolean }> = ({ isMobile }) => {
+  const [activeLink, setActiveLink] = useState("communities");
+
+  const handleLinkClick = (link) => {
+    setActiveLink(link);
+  }
 
   return (
     <section className="community__hero">
       {isMobile ? null : (
         <>
-          <h1>Communities</h1>
+          <div className="community__hero__toggle">
+              <h3>
+                <a className={`link ${activeLink === 'communities' ? 'active' : ''}`} onClick={() => handleLinkClick("communities")}>
+                  Communities
+                </a>
+              </h3>
+              <h3>
+                <a className={`link ${activeLink === 'initiatives' ? 'active' : ''}`} onClick={() => handleLinkClick("initiatives")}>
+                  Initiatives
+                </a>
+              </h3>
```

Comment:
change these to `<button>`s since they don't have URLs. this way the cursor becomes `pointer`